portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,75 @@
 <!DOCTYPE html>
 <html>
+    <style>
+    
+    body {
+    background-image: url('imgDir/black-close-up-dark-dew-396547.jpg');
+    background-repeat: no-repeat;
+    background-attachment: fixed; 
+    background-size: cover;
+    color: white;
+}
+    h3 {
+        font-family:verdana; 
+        text-align:center; 
+        color:white;
+        text-shadow: 3px 3px grey;
+        }
+    h5 {
+        font-family:verdana; 
+        text-align:left; 
+        color:white;
+        }
+    a:link{
+        color:blue;
+    }
+    a:visited{
+        color:red;
+    }
+    a:hover{
+        color:white;
+    }
+
+</style>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Yemi's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+      <img src="imgDir/myPic2.jpeg" alt="A picture of Yemi" style="width:400px;height:400px;">
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Yemi's Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
```

Comment:
If the addRandomGreeting method isn't being used, maybe remove if from webapp/script.js?